docs/source/ko/package_reference/inference_types.md

```diff
@@ -0,0 +1,345 @@
+
+<!--⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+-->
+<!--⚠️ Note that this file is auto-generated by `utils/generate_inference_types.py`. Do not modify it manually.-->
+
+
+# 추론 타입[[inference-types]]
+
+이 페이지에는 Hugging Face Hub에서 지원하는 타입(예: 데이터 클래스)이 나열되어 있습니다.
+각 작업은 JSON 스키마를 사용하여 지정되며, 이러한 스키마에 의해서 타입이 생성됩니다. 이때 Python 요구 사항으로 인해 일부 사용자 정의가 있을 수 있습니다.
+ 
+각 작업의 JSON 스키마를 확인하려면 [@huggingface.js/tasks](https://github.com/huggingface/huggingface.js/tree/main/packages/tasks/src/tasks)를 확인하세요.
+
+라이브러리에서 이 부분은 아직 개발 중이며, 향후 릴리즈에서 개선될 예정입니다.
+
+
+
+## audio_classification[[huggingface_hub.AudioClassificationInput]]
```

Comment:
Hi, @Wauplin !
Should we translate the task name (ex. `audio_classification`) in the heading or leave it as is?

Reply:
Good question. I think you can either keep them in English since these are "constrained" tags. Or have both the Korean + English versions in the same title like `audio-classification (오디오 분류) [[huggingface_hub.AudioClassificationInput]]` (disclaimer: translated with google translate).

Reply:
@fabxoe님은 둘 중에 어떤 방법으로 하는 게 좋다고 생각하시나요?

Reply:
@jungnerd 님,
결국에는 제 생각에는 그대로 두는것도 나쁘지 않아보입니다.

해당 부분의 원래 의도는 네이밍을 공유하는 비슷한 기능의 클래스들을 나열해 두고 네이밍의 공통 부분을 표현하기 위해서 대략 snake style로 작성된 것으로 보입니다. 

class huggingface_hub.DocumentQuestionAnsweringInput
class huggingface_hub.DocumentQuestionAnsweringInputData
class huggingface_hub.DocumentQuestionAnsweringOutputElement

이 클래스들의 타입명은 'document_question_answering' 로 단순한데 이대로 가독성이 좋습니다.
이것을 한글로 번역할 때 의미를 전달하고자 하는 의도였다면 '문서 질문 답변' 처럼 그대로 단순 번역이 되어 이해에 추가적인 도움이 되지는 않을 것 같습니다.
시각적인 의도라면 영어와 한글을 병기시 우측의 toctree의 메뉴의 너비도 커지지 않을까 합니다.

Reply:
좋아요! 성민님의 의견에 동의합니다 👍

Reply:
@Wauplin After your suggestion, we've been thinking about it and we think it's better to leave the task name untranslated. Thank you for the idea!

Reply:
Works for me as well. Thanks for iterating on it!